akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -199,6 +199,12 @@ Akka is now using Protobuf version 3.9.0 for serialization of messages defined b
 Cluster client has been deprecated as of 2.6 in favor of [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html).
 It is not advised to build new applications with Cluster client, and existing users @ref[should migrate to Akka gRPC](../cluster-client.md#migration-to-akka-grpc).
 
+### Persistent mode for Cluster Sharding
+
+Cluster Sharding coordinator and Replicated Entities state in 2.5 can be stored in Distributed Data or via Akka Persistence.
```

Comment:
it's not Replicated Entities, but "Remembering Entities"
maybe make that a link as well typed/cluster-sharding.md#remembering-entities